doc/sphinx/Pacemaker_Explained/cluster-options.rst

```diff
@@ -184,6 +184,16 @@ holds. So the decision was made to place them in an easy-to-find location.
      -
      - Node ID of the cluster's current designated controller (DC). Used and
        maintained by the cluster.
+   * - .. _execution_date:
+       
+       .. index::
+          pair: execution-date; cib
+       
+       execution-date
+     - :ref:`nonnegative integer <nonnegative_integer>`
+     -
+     - For saved input files, time (in seconds since epoch) when the input was
+       generated and the file was written to disk.
```

Comment:
In practice yes, but it's actually the time to use for evaluation of rules. Someone could set this to whatever they want (in a saved file) for testing purposes

Reply:
Oh neat :) I'll change it.

Reply:
commented on #3308 about not-exactly-correct value type: https://github.com/ClusterLabs/pacemaker/pull/3308#discussion_r1445302512

Not worried about it right now just FYI